utils/helpers.js

```diff
@@ -81,6 +81,17 @@ export const useFeatureCounts = ({ geojson, filters }) =>
     // eslint-disable-next-line
   }, [geojson]);
 
+/**
+ * Custom hook that sums the number of features that match the filters
+ * @param { object } featureCounts - An object with one prop per filter in the format { filter.key: count }
+ * @returns { Int } - sum of the counts
+ */
+export const useTotalFeatureCount = (featureCounts) =>
+  useMemo(() => {
+    if (!featureCounts) return;
```

Comment:
It looks like this whole page shows a spinner until the data request has returned, but it made me wonder what would display if there is a request failure and the second part of this hook never returned a count. I think it is good that it doesn't return 0 here since it would look like everything is normal when it wasn't. 💯 

Definitely an issue for another day!

Reply:
it fails before it gets here I think, getting the featureCounts fails and the entire page throws an error

Reply:
yes, definitely we are definitely lacking socrata error handling in this app. some typescript guardrails would also be really nice!